alerter/alerter.java

```diff
@@ -1,27 +1,29 @@
+package alerter;
+
 public class Alerter {
-    static int alertFailureCount = 0;
-    static int networkAlertStub(float celcius) {
-        System.out.println("ALERT: Temperature is " + celcius + " celcius");
-        // Return 200 for ok
+    
+
+  public static int alertFailureCount = 0;
+  static int networkAlertStub(float celcius) {

```

Comment:
stub and production code is the same file. They should be kept separate so that it helps in future maintainability and also test code can be easily excluded during shipment of software to field.